cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORParser.java

```diff
@@ -837,6 +978,9 @@ protected String _numberToName(int ch, boolean neg) throws IOException
                 // [dataformats-binary#269] (and earlier [dataformats-binary#30]),
                 // got some edge case to consider
                 if (i < 0) {
+                    if (isStringref) {
+                        _reportError("String reference index too large.");
```

Comment:
Please remove trailing `.`

Reply:
I think I got all the added errors, let me know if I missed any.